lib/update_remote_id.rb

```diff
@@ -0,0 +1,51 @@
+# Update events remote_id from existing schedule xml data
```

Comment:
Shouldn't this be a sidekiq-task which updates the remote-id when a new conferences / release is added?